### PR TITLE
Vote and Score

### PR DESCRIPTION
**Pull request by SwappyG**
- Added score during results
- added voting to determine of words are correct at the end

### CODE REVIEW

File: src/Score.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export const format_score_breakdown = (num_players) => {
+  switch (num_players) {
+    case 2: case 3:
```
Comment (Singh212dab) [inline src/Score.js:3]: case 2: case 3:
      return `\`13-24\` 🍓\n\`25-37\` 🍓🍓\n\`38-49\` 🍓🍓🍓\n\`50-62\` 🍓🍓🍓🍓\n\`63+\` 🍓🍓🍓🍓🍓`

break;
    case 4:
      return `\`16-31\` 🍓\n\`32-47\` 🍓🍓\n\`48-63\` 🍓🍓🍓\n\`64-79\` 🍓🍓🍓🍓\n\`80+\` 🍓🍓🍓🍓🍓`

break;
    case 5:
      return `\`19-38\` 🍓\n\`39-58\` 🍓🍓\n\`59-77\` 🍓🍓🍓\n\`78-97\` 🍓🍓🍓🍓\n\`98+\` 🍓🍓🍓🍓🍓`

break;
    case 6:
      return `\`23-45\` 🍓\n\`46-68\` 🍓🍓\n\`69-91\` 🍓🍓🍓\n\`92-114\` 🍓🍓🍓🍓\n\`115+\` 🍓🍓🍓🍓🍓`
  }
}

Comment (SwappyG) [reply src/Score.js:3]: @Singh212dab  Sorry i wasn't monitoring this, i'm not sure what your recommendation is. Are you suggesting adding `break` between the cases?

Comment (Singh212dab) [reply src/Score.js:3]: yes

Comment (SwappyG) [reply src/Score.js:3]: each `case` has a `return` statement, so `break` isn't needed

**Pull request merged**